### DISCUSSION TITLE
Kikked.json

### DISCUSSION THREAD

**Discussion by Kukushechka7-ezx** [Sokol-Games-GmbH/PixelCombats.PublicApi#99] [Ideas]
Опись в файле gamemode.json параметров для кика... Например:
`"kikked":
[
{ "prop": "jump", "value": 3.0 } ]`

**Comment by Kukushechka7-ezx** [comment]
@kkohno 
https://github.com/Kukushechka7-ezx/Kiked

**Comment by forwork10** [comment]
это не имеет смысла, умный читер не станет менять саму переменную, он залезет в сам байт код функции и вместо обращения к переменной подставит значение...

**Comment by kkohno** [comment]
не совсем понял зачем это? И давайте в дискуссию перенесу - там можно видеть что не прочитал еще...

**Reply by Kukushechka7-ezx** [reply]
Описал в файле, имя значения, и само значение. Для кика... Если у prop есть такое то значение то кикать
